src/js/inverted-index.js

```diff
@@ -0,0 +1,138 @@
+/* eslint class-methods-use-this: "off" */
+/* eslint no-unused-vars: "off" */
+
+/** Class representing a index. */
+class Index {
+  /**
+   * Create and Index and search the index.
```

Comment:
Does not seem like this JS-DOC is accurately describing what is going on in the constructor below.